### PR TITLE
Doc/Core#693 - Remove clean_entities attribute from CoreSection

### PR DESCRIPTION
**Pull request by trgiangdo**
https://github.com/Avaiga/taipy-core/issues/693

### CODE REVIEW

File: docs/manuals/core/config/core-config.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Here are the (optional) configurable properties:
   conjunction with the root_folder field. That means the default storage path is "./taipy/.data/".
 - _**read_entity_retry**_: An integer number only used with `filesystem` _**repository_type**_. <br>
 It corresponds to the number of times Taipy retries reading an entity after a failed attempt for concurrent access. <br>
-The default value is 3.
+The default value is 1.
 - _**repository_type**_: The type of storage that will be used to hold Taipy entities. Available options are:
   `filesystem`, `sql` and `mongo`. If no repository type is informed, the filesystem will be used as the default
   repository.
```
Comment (erikfTW) [inline docs/manuals/core/config/core-config.md]:
```suggestion
  `filesystem`, `sql` and `mongo`. The filesystem will be used as the default repository if no repository type is informed.
```

File: docs/manuals/core/versioning/configuration.md [hunk 2]
```diff
@@ -12,11 +12,11 @@ page.
 ```python linenums="1"
 from taipy import Config
 
-Config.configure_core(mode="experiment", version_number="0.1", clean_entities=True)
+Config.configure_core(mode="experiment", version_number="0.1")
 ```
 
 In the above example, we configure the version management system by setting the mode to experiment
-with the version number is "0.1", and clean all entities on each run.
+with the version number is "0.1".
```
Comment (erikfTW) [inline docs/manuals/core/versioning/configuration.md]:
```suggestion
with the version number "0.1".
```

File: docs/manuals/core/versioning/configuration.md [hunk 3]
```diff
@@ -80,10 +79,6 @@ configure the version management system:
 - **--taipy-force**: With the `--taipy-force` argument, Taipy overrides a version even if
   the configuration has changed and run the application. Default to False.
```
Comment (erikfTW) [inline docs/manuals/core/versioning/configuration.md]:
```suggestion
  the configuration has changed and runs the application. Default to False.
```

File: docs/manuals/core/versioning/configuration.md [hunk 4]
```diff
@@ -80,10 +79,6 @@ configure the version management system:
...
 
-- **--clean-entities**: With the `--clean-entities` argument, running a Taipy
-  Core application cleans all current version entities before running the application.
-  Default to False.
-
 !!! note "Configure using both TOML file and CLI options"
```
Comment (erikfTW) [inline docs/manuals/core/versioning/configuration.md]:
```suggestion
!!! note "Configure using both the TOML file and CLI options."
```

File: docs/manuals/core/versioning/experiment_mode.md [hunk 5]
```diff
@@ -102,35 +102,6 @@ Number of scenarios: 2
 As you can see, this time the number of scenarios displayed is 2. Indeed, we run the "0.1" version
```
Comment (erikfTW) [inline docs/manuals/core/versioning/experiment_mode.md]:
```suggestion
As you can see, this time, the number of scenarios displayed is 2. Indeed, we run the "0.1" version
```

File: docs/manuals/core/versioning/experiment_mode.md [hunk 6]
```diff
@@ -186,3 +157,21 @@ the configuration. A new scenario has been created.
 
     Hint: You can migrate your old entities so they become compatible with the new configuration or
     you can ensure your code is compatible with both versions of data nodes.
```
Comment (erikfTW) [inline docs/manuals/core/versioning/experiment_mode.md]:
```suggestion
    Hint: You can migrate your old entities to become compatible with the new configuration or
    ensure your code is compatible with both versions of data nodes.
```

File: docs/manuals/core/versioning/experiment_mode.md [hunk 7]
```diff
@@ -186,3 +157,21 @@ the configuration. A new scenario has been created.
...
+
+# Delete a version
+
+To delete an experiment version, you can use the `taipy manage-versions` with the `--delete`
+option on the Taipy CLI and provide the version name (see the
+[Manage versions on Taipy CLI page](../../cli/manage-versions.md) for more details).
+
+```console
+$ taipy manage-versions --delete 0.1
+Successfully delete version 0.1 from the production version list.
```
Comment (erikfTW) [inline docs/manuals/core/versioning/experiment_mode.md]:
```suggestion
Successfully deleted version 0.1 from the production version list.
```

**Review by erikfTW — CHANGES_REQUESTED** [review]

**Review by jrobinAV — APPROVED** [review]

File: docs/relnotes.md [hunk 8]
```diff
@@ -55,6 +55,10 @@ Not published yet.
 - The `root_folder`, `storage_folder`, `read_entity_retry`, `repository_type`, and `repository_properties`
   attributes of the `GlobalAppConfig^` have been moved to the `CoreSection^`.<br/>
   Please refer to the [Core configuration page](core-config.md) for details.
+- The `clean_entities` attribute has been removed from the `CoreSection^`. Correspondingly, the
```
Comment (jrobinAV) [inline docs/relnotes.md:58]:
```suggestion
- The `clean_entities` attribute has been removed from the `GlobalAppConfig^`. Correspondingly, the
```

**Review by jrobinAV — APPROVED** [review]

File: docs/relnotes.md (hunk 8 above)
Comment (trgiangdo) [reply docs/relnotes.md:58]: the `clean_entities` attribute belongs to CoreSection, not the GlobalAppConfig though.

**Review by erikfTW — APPROVED** [review]

File: docs/relnotes.md (hunk 8 above)
Comment (jrobinAV) [reply docs/relnotes.md:58]: In previous versions like 2.4 the clean_entities was part of GlobalAppConfig, no?

Comment (trgiangdo) [reply docs/relnotes.md:58]: Actually no, it was part of the CoreSection.

Comment (jrobinAV) [reply docs/relnotes.md:58]: OK, I must be confused... Let s forget about my suggestion then.

**Pull request merged**